test/node/reducers/vis-state-test.js

```diff
@@ -2478,7 +2478,7 @@ test('#visStateReducer -> REMOVE_DATASET w filter and layer', t => {
     splitMaps: oldState.splitMaps,
     layerClasses: oldState.layerClasses,
     animationConfig: oldState.animationConfig,
-    initialState: [],
+    initialState: oldState.initialState,
```

Comment:
Tests were failing because `[]` was expected and the actual state was `{}`, not sure how my changes affected this, but `{}` seems like the right value?